mimic/model/heat_objects.py

```diff
@@ -135,6 +135,8 @@ def creation_response_json(self, absolutize_url):
 
 
 stack_creation = EventDescription()
+stack_check = EventDescription()
```

Comment:
It feels to me like removing the behaviors here would be moving backwards, and we should instead be adding in error behaviors and a control plane.